histolab/masks.py

```diff
@@ -134,8 +135,17 @@ def _regions(regions: List[Region], n: int = 1) -> List[Region]:
 class TissueMask(BinaryMask):
     """Object that represent the whole tissue area mask.
 
-    The tissue within the slide or tile is automatically detected through a predefined
-    chain of filters."""
+    By default, the tissue within the slide or tile is automatically
+    detected through a predefined chain of filters.
+    However, custom filters could be used, too.
+    """
+
+    def __init__(self, *filters: image_filters.ImageFilter):
+        """
+        Create a new tissue mask. If custom image filters are specified,
+        those are used instead of the default ones.
```

Comment:
IMHO considering the https://www.python.org/dev/peps/pep-0257/ the first line of the docstring should be a sort of summary and fit in 88 chars. All the other info can be underneath the summary line with a black line between them.

Moreover the `Parameters` section of the docstring is missed 😄